core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SampleClass.java

```diff
@@ -4,68 +4,65 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
-import java.util.Date;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
-import com.eaglegenomics.simlims.core.User;
+import uk.ac.bbsrc.tgac.miso.core.util.MapBuilder;
 
-public interface SampleClass extends Identifiable, Serializable {
+public interface SampleClass extends Deletable, Identifiable, Serializable, Timestamped {
 
-  List<String> CATEGORIES = Collections.unmodifiableList(Arrays.asList(SampleIdentity.CATEGORY_NAME, SampleTissue.CATEGORY_NAME,
-  SampleTissueProcessing.CATEGORY_NAME, SampleStock.CATEGORY_NAME, SampleAliquot.CATEGORY_NAME));
+  public static final List<String> CATEGORIES = Collections.unmodifiableList(Arrays.asList(SampleIdentity.CATEGORY_NAME,
+      SampleTissue.CATEGORY_NAME, SampleTissueProcessing.CATEGORY_NAME, SampleStock.CATEGORY_NAME, SampleAliquot.CATEGORY_NAME));
 
-  String getAlias();
+  public static final Map<String, List<String>> SUBCATEGORIES = new MapBuilder<String, List<String>>()
+      .put(SampleIdentity.CATEGORY_NAME, SampleIdentity.SUBCATEGORIES)
+      .put(SampleTissue.CATEGORY_NAME, SampleTissue.SUBCATEGORIES)
+      .put(SampleTissueProcessing.CATEGORY_NAME, SampleTissueProcessing.SUBCATEGORIES)
+      .put(SampleStock.CATEGORY_NAME, SampleStock.SUBCATEGORIES)
+      .put(SampleAliquot.CATEGORY_NAME, SampleAliquot.SUBCATEGORIES)
+      .build();
 
-  void setAlias(String alias);
+  public String getAlias();
 
-  User getCreatedBy();
+  public void setAlias(String alias);
 
-  void setCreatedBy(User createdBy);
+  public String getSampleCategory();
 
-  Date getCreationDate();
+  public void setSampleCategory(String sampleCategory);
 
-  void setCreationDate(Date creationDate);
+  public String getSampleSubcategory();
 
-  User getUpdatedBy();
-
-  void setUpdatedBy(User updatedBy);
-
-  Date getLastUpdated();
-
-  void setLastUpdated(Date lastUpdated);
-
-  String getSampleCategory();
-
-  void setSampleCategory(String sampleCategory);
-
-  String getSampleSubcategory();
-
-  void setSampleSubcategory(String sampleCategory);
+  public void setSampleSubcategory(String sampleCategory);
 
   /**
    * @return the class identifier used in Sample alias generation
    */
-  String getSuffix();
+  public String getSuffix();
 
   /**
    * Sets the class identifier used in Sample alias generation
    * 
    * @param suffix
    */
-  void setSuffix(String suffix);
+  public void setSuffix(String suffix);
+
+  public boolean isArchived();
+
+  public void setArchived(boolean archived);
 
-  boolean isArchived();
+  public boolean isDirectCreationAllowed();
 
-  void setArchived(boolean archived);
+  public void setDirectCreationAllowed(boolean directCreationAllowed);
 
-  boolean isDirectCreationAllowed();
+  public boolean getDNAseTreatable();
```

Comment:
It looks like everything else is using 'is__' for boolean getters, is there a reason this method (and others named the same thing) still use 'get'?

Reply:
It hadn't annoyed me enough to rename it? We don't consistently use one or the other throughout MISO

Reply:
Alright! Just noticed getArchived() became isArchived() and this one didn't :+1:

Reply:
`getArchived` had annoyed me enough 😆